retry.go

```diff
@@ -22,6 +24,62 @@ func init() {
 	random = rand.New(rand.NewSource(time.Now().UnixNano()))
 }
 
+const retryKey string = "request_guid"
```

Comment:
can you change the name to `equestGUID`
retry key is a separate one and not part of this change (and only JDBC has the retry parameter...)